__tests__/app.test.js

```diff
@@ -40,4 +41,16 @@ describe('GET /api/topics', () => {
             }
         });
     });
+});
+
+
+describe('GET /api', () => {
+    test('returns contents of endpoints.json', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then((response) => {
+            expect(JSON.parse(response.text)).toEqual(endpoints);
```

Comment:
Nice, but similar feedback to the previous task, there is a better way to access the response data, please do refer back to the notes for this.